plugins/com.gwtplugins.gdt.eclipse.suite.update/src/com/google/gdt/eclipse/suite/update/usage/AnalyticsPingManager.java

```diff
@@ -169,7 +168,7 @@ private static String getParametersString(Map<String, String> parametersMap) {
       }
       resultBuilder.append(entry.getKey());
       resultBuilder.append('=');
-      resultBuilder.append(ESCAPER.escape(entry.getValue()));
+      resultBuilder.append(URLEncoder.encode(entry.getValue(), Charsets.UTF_8));
```

Comment:
could also be `java.nio.charset.StandardCharsets.UTF_8`

Reply:
Yes!